msteams-platform/concepts/build-and-test/share-to-teams-from-web-apps.md

```diff
@@ -49,46 +62,227 @@ See the following video to learn how to embed Share to Teams button:
+
+Before you enable share content to meetings, you must ensure the following:
+
+1. A [Microsoft Teams app with a tab](../../apps-in-teams-meetings/build-tabs-for-meeting.md) that supports meeting side panel and Share to Stage APIs.
+
+2. To display the content on the meeting stage, the meeting app must support [Share to Stage APIs](../../apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md) with two primary requirements:
+
+    * The [app manifest](../../resources/schema/manifest-schema.md) that supports both `meetingStage` and `meetingSidePanel` configured as frame contexts. If not configured, meeting participants might not be able to view the content on stage.
+    * The app that supports `MeetingStage.Write.Chat` permissions for taking control of the stage. This is a read permission requirement.
+
+> [!NOTE]
+>
+> * If the app doesn't have meeting stage and side panel capabilities, the content aren't stored and the messages are sent in the regular meeting chat.
+> * If app installation is blocked by the admin, then the app is added to the chat and not the meeting of the user.
+> Share to Teams in meeting is supported only for instant tab scenarios and specific to Teams.
+
+**Enable Share to Teams for meeting**
+
+1. Add the `launcher.js` script on your webpage.
 
     ```html
-    <div
-      class="teams-share-button"
-      data-href="https://<link-to-be-shared>"
-      data-preview="false">
-    </div>
+    <script async defer src="https://teams.microsoft.com/share/launcher.js"></script>
     ```
 
-1. To display a message of your choice in compose box, you can define your text in `data-msg-text` attribute.
+1. Add an HTML element on your webpage with the `teams-share-button` in the `class` attribute and the link to share in the `data-href` attribute. To enable your users to share content in meetings from Share to Teams, add `allow-share-in-meeting` attribute and set it to `true`.
 
-     ```html
-     <div
-      class="teams-share-button"
-      data-href="https://<link-to-be-shared>"
-      data-msg-text="<default-message-to-be-populated-in-compose-box>"
-      data-preview="false">
-      </div>
-     ```
+    ```html
+        <div
+        class="teams-share-button"
+        data-href="https://<link-to-be-shared>"
+        data-allow-share-in-meeting="true"
+        data-app-id="<app-id>"
+        >
+        </div>
+    ```
+
+    After the configuration, the Share to Teams dialog appears with a **Present now** option. Present now allows the users to share content to an ongoing meeting.
+
+    :::image type="content" source="~/assets/images/share-to-teams-popup.png" alt-text="Screenshot shows the Share to Teams dialog with Present now option.":::
+
+Here are a few end user scenarios where the content is shared to different scopes from Share to Teams.
+
+</br>
+<details>
+<summary>Share content to a chat or channel</summary>
```

Comment:
Do we need this? This section is only for Share to Meetings right?

Reply:
Yes, we can remove. Updated.